src/assets/Timecard.scss

```diff
@@ -3,3 +3,61 @@
     margin-top: 20px;
   }
 }
+
+@media only screen and (max-width: 640px) {
+  .mobile-link-group {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    margin-bottom: 0%;
+  }
+
+  .govuk-caption-m {
+    margin-top: 2%;
+  }
+
+  .mobile-checkbox {
+    margin-top: -5%;
+  }
+
+  .mobile-heading {
+    margin-top: 0%;
```

Comment:
This only seems to appear next to the use of `govuk-caption-m` which is only used once. Why would we not just update this  `govuk-caption-m` style